test/fixtures/users.yml

```diff
@@ -115,3 +115,27 @@ system_admin:
   auth_source: internal
   password_hash: 02d7ff9921071af778ff4f8608579dcd6d80dfba
   password_salt: 80a167f1effbd82c2485ed81c3cfd68b11bc40dc
+
+disabled:
```

Comment:
I don't like fixtures for just one test :thinking: cat we take another user, e.g. `one`?